apps/charge-api-service/src/paymaster-api/paymaster-api.service.ts

```diff
@@ -29,8 +29,13 @@ export class PaymasterApiService {
   async pm_sponsorUserOperation (body: any, env: any, projectId: string) {
     try {
       const web3 = this.paymasterWeb3ProviderService.getProviderByEnv(env)
+      console.log(web3.currentProvider)
```

Comment:
that can give a pretty long response.

Reply:
Ill delete it, since we didnt need this anymore